modules/ROOT/pages/edifact-receive-read-settings.adoc

```diff
@@ -0,0 +1,147 @@
+|A transaction with a segment repeated too many times is rejected. The error is reported in the 997 or 999 acknowledgments.
+|Enabled
+
+|Fail document when unused segments are included
+|If selected, a transaction containing segments marked as unused in the schema is rejected. If unselected, the transaction is not rejected and the unused segments are ignored. In either case, the error is reported in the 997 or 999 acknowledgments.
+|Enabled
+|===
+
+[[control-numbers]]
+=== Control Numbers
+
+Use these settings to apply validations related to your partners or your use of control numbers within the EDIFACT message:
+
+[%header%autowidth.spread]
+|===
+|Setting |Description |Default
+
+|Requires unique interchange control number (ISA13)
+|If selected, Partner Manager records the interchange numbers previously processed and rejects any duplicate interchange numbers from the same partner (as determined by the interchange sender and receiver identification). If unselected, it enables processing of the received interchange to continue.
+|Disabled
+
+|Requires unique group control number (GS06)
+|If selected, Partner Manager enforces globally unique Group Control Numbers (GS06) for received functional groups. This configuration requires group numbers to be unique across all interchanges received from the same partner and application (as determined by the interchange sender and receiver identification, combined with the functional group sender and receiver application codes).
+| Disabled
+
+|Requires unique transaction set control number (ST02)
+|If selected, Partner Manager enforces globally unique Transaction Set Control Numbers (ST02) for received transaction sets. This configuration requires transaction set numbers to be unique across all functional groups received from the same partner and application (as determined by the interchange sender and receiver identification, combined with the functional group sender and receiver application codes).
+|Disabled
+|===
+
+[[character-sets]]
+=== Character Sets
+
+Use these settings to apply selected character set and encoding options pertaining to your partners and your EDIFACT message:
+
+[%header%autowidth.spread]
+|===
+|Setting |Description |Default
+
+|Character set
+a|This option is required and defines the characters allowed in string data. When set, invalid characters are replaced by the substitution character. If no substitution character is set or enabled for receive messages in the parser options, they are rejected as errors.
+Either way, the invalid characters are logged and are reported in the 997 functional acknowledgments for the receive messages.
+
+Possible values include:
+
+* `Basic`
+* `Extended`
+* `Unrestricted`
+| `Extended`
+
+|Character encoding
+a|Indicates the character encoding for messages. This character encoding is used for both send and receive messages.
+
+Possible values include:
+
+* `ASCII`
+* `ISO8859_1`
+* `UTF-8`
+|Disabled
+|===
```

Comment:
@dhirsch1 - This topic needs to be replaced with the content from https://docs.google.com/document/d/18s4ohdBeGkHkwDS1EDWAEyEfHFsNFbMFExnw-RXVKcY/edit